Czaple/Models/FirstLaunch.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FirstLaunch.swift
+//  Czaple
+//
+//  Created by Filip Trzaska on 05/12/2019.
+//  Copyright © 2019 Filip Trzaska. All rights reserved.
+//
+
+import UIKit
+
+class FirstLaunch {
+    static let instance = FirstLaunch()
+    
+    func checkFirstLaunch(window: UIWindow){
```

Comment:
I would suggest to split the logic. You should have one class responsible for checking and setting thins in user defaults and other that decides what should be presented based on that.